src/frontend/src/styleguide.ts

```diff
@@ -278,6 +293,29 @@ export const styleguide = html`
         </p>
       </aside>
 
+
+      <aside class="l-stack demo-section">
+        <h2 class="t-title t-title--sub">Recovery Word List</h2>
+        <section class="demo" aria-label="Recovery List Elements Demo">
+          <output class="c-input c-input--textarea c-input--readonly c-input--icon">
+            <ol class="c-list c-list--recovery">
+              ${recoveryWords.map(
+                (word, i) =>
+                  html`<li style="--i: ${i / recoveryWords.length}">
+                    ${word}
+                  </li>`
+              )}
+            </ol>
+            <i aria-label="Copy phrase to clipboard" "="" title="Copy phrase to clipboard" tabindex="0" id="seedCopy" class="c-button__icon c-input__icon">
+            <span>Copy</span>
+            <svg width="18" height="22" viewBox="0 0 18 22" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Should we extract this like the other icons? Looking at this I'm not sure what this does

Reply:
I did not change that, let do this in an other branch.

Reply:
Most designs I found are RTL: [recovery phrase design](https://www.google.com/search?q=recovery+phrase+design&sxsrf=AJOqlzWet1gHO6mQWnJNhHKCmq-W9aPKhw:1676895723924&source=lnms&tbm=isch&sa=X&ved=2ahUKEwjrrcyvi6T9AhUQxwIHHScXCToQ_AUoAXoECCcQAw&biw=1512&bih=808&dpr=2#imgrc=kXZX7_5MpK-deM) 

But I don't recall any user tests or UX reads about this topic so we can change it if you feel strongly about it